Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift

```diff
@@ -394,3 +406,16 @@ class PhotonActionSheet: UIViewController, UITableViewDelegate, UITableViewDataS
         return view
     }
 }
+
+extension PhotonActionSheet: PhotonActionSheetCollectionCellDelegate {
+
+    func collectionCellDidSelectItem(item: PhotonActionSheetItem) {
+        guard let handler = item.handler else {
+            self.dismiss(nil)
```

Comment:
dismiss is called in all cases so can be moved out of guard

Reply:
Done.